core/google/cloud/_helpers.py

```diff
@@ -550,7 +549,7 @@ def make_secure_channel(credentials, user_agent, host, extra_options=()):
     :returns: gRPC secure channel with credentials attached.
     """
     target = '%s:%d' % (host, http_client.HTTPS_PORT)
-    http_request = google_auth_httplib2.Request(http=httplib2.Http())
+    http_request = google.auth.transport.requests.Request()
```

Comment:
I don't remember the reasoning, but without an argument it'll just create its own `requests.Session`.